src/main/java/bit/javaoop/CEO.java

```diff
@@ -0,0 +1,34 @@
+package bit.javaoop;
+
+import java.util.LinkedList;
+
+public class CEO {
+    private final String name;
+    private LinkedList<Manager> managers;
```

Comment:
- Jest przyjęta taka zasada, że deklarujemy najpierw stałe a potem od najbardziej prymitywnych obiektów czyli salary powinno być nad name
- Staramy się zawsze wykorzystywać jak najwyższy poziom abstrakcji jaki odpowiada naszym potrzebom tutaj moglibyśmy wykorzystać interfejs List daje nam to dużo większe flexibility w wykorzystywanej implementacji, gdybyśmy potem chcieli zmienić implementacje wystarczyłoby zrobić to w jednym miejscu czyli powinno być tak:
`private List<Manager> managers;`
- Co więcej na ten moment ArrayLista jest tak zoptymalizowana, że właściwie nie ma sensu używać innej implementacji listy ^^